src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DataFlowOperationVisitor.cs

```diff
@@ -990,8 +990,11 @@ private bool IsContractCheckMethod(IMethodSymbol method)
             return false;
         }
 
-        private bool IsDebugAssertMethod(IMethodSymbol method)
-            => Equals(method, DebugAssertMethod);
+        private bool IsDebugAssertMethod(IMethodSymbol method) =>
+            DebugAssertMethod != null &&
```

Comment:
`DebugAssertMethod` property initialized with `Debug.Assert(bool)` method if exist

Reply:
Can you please add a doc comment and also rename the method to IsAnyDebugAssertMethod?

Reply:
Sure